scripts/s3-support-bundles/s3_bundle_generate.sh

```diff
@@ -38,8 +38,17 @@ fi
 
 bundle_id=$1
 bundle_path=$2
-# TODO: till we find out how to get rootdn password, keep 'rootdnpasswd' variable empty.
+# Fetch iamuser password from properties file and decrypt it.
 rootdnpasswd=''
+constkey=cortx
+propertiesfilepath="properties:///opt/seagate/cortx/auth/resources/authserver.properties"
+ldapkey="ldapLoginPW"
+ldapcipherkey=$(s3cipher generate_key --const_key $constkey)
```

Comment:
if s3cipher, s3confstore fails then we should skip ldap collection .
Support bundle should be working even in case of s3confstore,s3cipher command failures.
cc @AjinkyaDhumal , @bkirunge7

Reply:
if s3cipher or s3confstore fails the support bundle collection does not fail.
Please check output of Case 2 and 3 mentioned in PR Summary

Reply:
one minor suggestion .. can we hide these error stack msgs from console..this creates confusion and people will think that something has broken in support bundle ..
e.g. 
(cmd) 2> /dev/null || echo "INFO : s3cipher failed to decrypt ldap password hence skipping ldap data collection"